src/sort_transform.c

```diff
@@ -272,7 +272,9 @@ sort_transform_ec(PlannerInfo *root, EquivalenceClass *orig)
 			 */
 			EquivalenceClass *exist = get_eclass_for_sort_expr(root,
 															   transformed_expr,
+#if PG16_LT
 															   ec_mem->em_nullable_relids,
+#endif
 															   opfamilies,
 															   type_oid,
 															   orig->ec_collation,
```

Comment:
Even though this works since `get_eclass_for_sort_expr` is not a macro, I would avoid preprocessor macros inside parameter lists.